bin/cml/comment/create.js

```diff
@@ -0,0 +1,53 @@
+const kebabcaseKeys = require('kebabcase-keys');
+
+exports.command = 'create <markdown file>';
+exports.description = 'Create a comment';
```

Comment:
To my mind, CML messaging about reports has to be backed by a technical `report` feature that simplifies comments and checks. Otherwise, it's going to be rather misleading.